frontend/src/components/CytoscapeGraph/MiniGraphCard.tsx

```diff
@@ -76,6 +77,9 @@ class MiniGraphCard extends React.Component<MiniGraphCardProps, MiniGraphCardSta
   }
 
   private refresh = () => {
+    if (isKioskMode()) {
+      this.props.dataSource.graphDuration = store.getState().userSettings.timeRange.rangeDuration!;
+    }
```

Comment:
I don't think this should be done, as data has already been fetched at this point. The `graphDuration` should already hold the right value of the already fetched data.